system_metrics_collector/src/topic_statistics_collector/subscriber_topic_statistics.hpp

```diff
@@ -301,12 +316,17 @@ class SubscriberTopicStatisticsNode : public system_metrics_collector::MetricsMe
   /**
    * Subscriber to listen to incoming messages on a topic
    */
-  typename rclcpp::Subscription<T>::SharedPtr subscription_;
+
+
+  //typename rclcpp::Subscription<T>::std::vector<SharedPtr> subscription_;
+  std::vector<typename rclcpp::Subscription<T>::SharedPtr> subscriptions_;
+  // typename rclcpp::Subscription<T>::SharedPtr subscription_;
 
   /**
    * Topic name to compute statistics for
    */
-  std::string collect_topic_name_;
+  // std::string collect_topic_names_;
```

Comment:
Remove?

Reply:
Since this feature is not yet complete, I thought of keeping those comments. But yes, will remove them.